janusgraph-es/src/main/java/org/janusgraph/diskstorage/es/ElasticSearchIndex.java

```diff
@@ -727,12 +760,23 @@ public void mutate(Map<String, Map<String, IndexMutation>> mutations, KeyInforma
                                 upsert = null;
                             }
 
-                            final String inline = getAdditionScript(information, storeName, mutation);
-                            if (!inline.isEmpty()) {
-                                final ImmutableMap.Builder builder = compat.prepareScript(inline);
-                                requestByStore.add(ElasticSearchMutation.createUpdateRequest(indexStoreName, storeName,
-                                        documentId, builder, upsert));
-                                log.trace("Adding script {}", inline);
+                            if (compat.supportsParams()) {
+                                List<Map<String, Object>> params = getParameters(information.get(storeName),
+                                        mutation.getAdditions(), false, Cardinality.SINGLE);
+                                if (!params.isEmpty()) {
+                                    ImmutableMap.Builder builder = compat.prepareScript(PARAMETERIZED_ADDITION_SCRIPT, params);
+                                    requestByStore.add(ElasticSearchMutation.createUpdateRequest(indexStoreName, storeName,
+                                            documentId, builder, upsert));
+                                    log.trace("Adding script {} with params {}", PARAMETERIZED_ADDITION_SCRIPT, params);
+                                }
+                            } else {
+                                final String inline = getAdditionScript(information, storeName, mutation);
```

Comment:
Same here. We can just drop the whole logic inside `else`. We don't need to support ES version less than 6 anymore

Reply:
done